src/usbhostmanager.cpp

```diff
@@ -10,8 +10,12 @@ void USBHostManager::setDataPin(uint8_t inPin) {
 
 void USBHostManager::start() {
     if ( !addons.empty() ) {
+    	// State Machine moved to PIO0:1 and PIO1:1,2 for NeoPico
         pio_usb_configuration_t pio_cfg = PIO_USB_DEFAULT_CONFIG;
         pio_cfg.pin_dp = dataPin;
+        pio_cfg.sm_eop = 2;
+        pio_cfg.sm_rx = 1;
+        pio_cfg.sm_tx = 1;
```

Comment:
NeoPico utilizes only pio0:0, so reconfiguring tx should be enough.

Reply:
Yeah, that makes sense. I'll revert back eop and rx